src/packages/verticalsteps/doc.zh-TW.md

```
@@ -0,0 +1,98 @@
+| title | 流程步驟的標題 | `string` | `-` |
+| description | 流程步驟的描述性文字 | `string` | `-` |
+| icon | 圖標(來自Icon組件的name屬性) | `ReactNode` | `-` |
+| value | 流程步驟的索引 | `number` | `0` |
+| description | 流程步驟的描述性文字的html結構 | `React.ReactNode` | `-` |
+| onStepClick | 點擊步驟的標題或圖標時觸發 | `(index: number) => void` | `-` |
+
+## 主題定制
+
+### 樣式變量
+
+組件提供了下列 CSS 變量，可用於自定義樣式，使用方法請參考 [ConfigProvider 組件](#/zh-CN/component/configprovider)。
```

Markdownlint

Link fragments should be valid

(MD051, link-fragments)